doc/src/build/move/index.md

```diff
@@ -2,89 +2,50 @@
+
+In Sui, Move is used to define, create, and manage programmable [Sui objects](../objects.md) representing user-level assets. Sui's object system is implemented by adding new functionality to Move while also imposing additional restrictions, creating a dialect of Move (a.k.a. *Sui Move*) that
+makes certain parts of the original Move documentation not applicable to smart contract development in Sui. Consequently, it's best to follow this tutorial and the relevant Move documentation links within.
+
+Before looking at the Move code included with Sui, let's talk briefly about Move code organization, which applies both to code included with
+Sui and the custom code developers write.
 
 
 ## Move code organization
 
-The main unit of Move code organization (and distribution) is a
-_package_. A package consists of a set of _modules_ defined in separate
-files with the `.move` extension. These files include Move functions and
-type definitions. A package must include the `Move.toml` manifest file
-describing package configuration, for example package metadata or
-package dependencies. See
-[Move.toml](https://github.com/move-language/move/blob/main/language/documentation/book/src/packages.md#movetoml)
-for more information about package manifest files.
+The main unit of Move code organization (and distribution) is a _package_. A package consists of a set of _modules_ defined in separate
+files with the `.move` extension. These files include Move functions and type definitions. A package must include the `Move.toml` manifest file
+describing package configuration, such as package metadata and package dependencies. See [Move.toml](manifest.md) for more information about package manifest files. Packages also include an auto-generated `Move.lock` file. The `Move.lock` file is similar in format to the package manifest, but is not meant for users to edit directly. See [Move.lock](lock-file.md) for more information about the lock file. 
 
-The minimal package source directory structure looks as follows and
-contains the manifest file and the `sources` subdirectory where one or
-more module files are located:
+The minimal package source directory structure looks as follows and contains the manifest file, the lock file, and the `sources` subdirectory where one or more module files are located:
 
 ```
 my_move_package
+├── Move.lock
 ├── Move.toml
 ├── sources
     ├── my_module.move
 ```
 
-See
-[Package Layout and Manifest Syntax](https://github.com/move-language/move/blob/main/language/documentation/book/src/packages.md#package-layout-and-manifest-syntax)
-for more information on package layout.
+See [Package Layout and Manifest Syntax](https://github.com/move-language/move/blob/main/language/documentation/book/src/packages.md#package-layout-and-manifest-syntax) for more information on package layout.
```

Comment:
wondering whether we should update all links from Move repo to Sui docs, such as https://docs.sui.io/devnet/build/move/write-package for package info here.